CHANGELOG.md

```diff
@@ -3,6 +3,23 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [vector-0.38.0] - 2024-12-04
```

Comment:
This diff doesn't seem right. It's not what I get when running `./.github/release-changelog.sh` on `develop`, which is instead:

```diff
+## [vector-0.38.0] - 2024-12-04
+
+### Vector
+
+#### Features
+
+- Bump Vector to v0.43.0 (#430) ([630594c](https://github.com/vectordotdev/helm-charts/commit/630594cbbc1051d8a8fd1686249173882a91ad3a))
```

Reply:
Hmm 

```
❯ ls
CHANGELOG.md  LICENSE  README.md  cat  charts  kubeval
❯ ./.github/release-changelog.sh
++ yq eval .version charts/vector/Chart.yaml
+ _chart_version=0.38.0
+ git cliff --config .github/cliff.toml --tag vector-0.38.0 --prepend CHANGELOG.md --unreleased
❯ git diff | cat
diff --git a/CHANGELOG.md b/CHANGELOG.md
index 794cbb0..65a62fb 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,6 +3,15 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [vector-0.38.0] - 2024-12-04
+
+### Vector
+
+#### Features
+
+- Bump Vector to v0.43.0 (#430) ([630594c](https://github.com/vectordotdev/helm-charts/commit/630594cbbc1051d8a8fd1686249173882a91ad3a))
+- Bump Vector to v0.43.0 ([ac6bbcb](https://github.com/vectordotdev/helm-charts/commit/ac6bbcb216162e4d404ca79ac6901e2f8e778adf))
+
 ## [vector-0.37.0] - 2024-12-04
 
 ### Vector
```

```
commit ac6bbcb216162e4d404ca79ac6901e2f8e778adf (HEAD -> develop)
Author: Pavlos Rontidis <pavlos.rontidis@gmail.com>
Date:   Wed Dec 4 11:51:39 2024 -0500

    feat(vector): Bump Vector to v0.43.0
```